### DISCUSSION TITLE
Relative number and current line

### DISCUSSION THREAD

**Discussion by mestrellar28** [LazyVim/LazyVim#3561] [Q&A] [answered]
Hello, why the current line is highlighted in orange and does not have the right margin?

How can i add this right margin?

Thanks!

<img width="1366" alt="SCR-20240610-jzpr" src="https://github.com/LazyVim/LazyVim/assets/41121749/f2afc17b-b860-4c86-8aaa-f1932de41fa1">

<img width="341" alt="SCR-20240610-kakf" src="https://github.com/LazyVim/LazyVim/assets/41121749/b874c69f-255c-4f62-9dd6-984566df75dd">

**Comment by dpetka2001** [comment, marked as answer]
The right margin comes from the custom LazyVim `statuscolumn`. You can set in your `/lua/config/options.lua` the value `vim.opt.statuscolumn = ""` to get default Neovim `statuscolumn`. The highlight comes from the colorscheme itself. It's the `CursorLineNr` highlight group that defines the color.

**Reply by mestrellar28** [reply]
That's it.. Thank you!
